### ISSUE TITLE
New Oauth requirements

### ISSUE THREAD

**Issue by richrosa** [fabriceb/sfFacebookConnectPlugin#24]
Has this plugin been rendered outdated by the new FB Oath  requirements?

**Comment by dlondero** [comment]
Yes, it is not working right now. It needs some changes, I hope to see them asap.

**Comment by richrosa** [comment]
I looked yesterday but didn't see any activity in any of the branches. I'm not looking forward to learning how this works. I hope someone who knows this code better than me has a quick solution.

**Comment by chok** [comment]
For a quick solution you can use sfMelodyPlugin. It is OAuth Based so not only for Facebook but Twitter, Google.... 

You can also try to contribute to sfFacebookConnectPlugin by forking and make a pull request ;-)

**Comment by richrosa** [comment]
One of the branches has a fix. You need to update php-sdk and make a few changes to the code. I'm going to fork the theodo branch for I have some changes that relate to syncing an existing sfGuardUser with FB credentials and allowing them to login as either a FB user of a sfGuard user.

**Comment by richrosa** [comment]
Is thodo branch working now?

**Comment by dlondero** [comment]
I don't know. Ended up using directly facebook-php-sdk and a couple of new methods just to sign in and create new users.
